src/main/java/org/spongepowered/common/data/DataRegistrar.java

```diff
@@ -762,9 +770,7 @@ public static void setupSerialization() {
         DataUtil.registerValueProcessor(Keys.VANISH, new VanishValueProcessor());
         DataUtil.registerValueProcessor(Keys.VANISH_IGNORES_COLLISION, new VanishCollisionValueProcessor());
         DataUtil.registerValueProcessor(Keys.VANISH_PREVENTS_TARGETING, new VanishTargetValueProcessor());
-        DataUtil.registerValueProcessor(Keys.DYE_COLOR, new WolfDyeColorValueProcessor());
-        DataUtil.registerValueProcessor(Keys.DYE_COLOR, new SheepDyeColorValueProcessor());
```

Comment:
Where do these go now?

Reply:
Aren't they registered in https://github.com/SpongePowered/SpongeCommon/blob/bleeding/src/main/java/org/spongepowered/common/data/util/DataUtil.java#L298 ?

Reply:
Ah yeah, subtle name change from value to data processor. good catch!